music21/_version.py

```diff
@@ -50,7 +50,7 @@
 '''
 from __future__ import annotations
 
-__version__ = '9.7.2a4'
+__version__ = '9.7.2a6'
```

Comment:
little thing since you're changing this a lot.  Call it version '9.8.0' and then '9.8.1.' etc. -- we use even minor digits to signify betas/not-released so we don't need 'alpha' or 'beta' on it.

(Plus this isn't what I call alpha -- which means we're pretty sure this code doesn't work -- this should be beta at lowest, since we're merging to master).

Reply:
Ah, I was trying real hard not to burn numbers.  I will do as you say.

Reply:
Done.  Switched to 9.7.3 for this PR.